packages/ui/src/contexts/shortcuts.tsx

```diff
@@ -0,0 +1,51 @@
+import {ComponentChildren, createContext} from 'preact';
+import {useContext, useState} from 'preact/hooks';
+
+export type Shortcut = {
+  key: string;
+  action: string;
+  isGlobal?: boolean;
+};
+
+export type ShortcutModules = 'global' | 'timeline' | 'none';
+type ShortcutsByModule = Record<ShortcutModules, Shortcut[]>;
+type ShortcutsState = {
+  currentModule: ShortcutModules;
+  shortcuts: ShortcutsByModule;
+  setCurrentModule?: (module: ShortcutModules) => void;
+};
+
+const initialshortcuts: ShortcutsByModule = {
+  global: [
+    {key: 'Space', action: 'Toggle playback', isGlobal: true},
+    {key: '←', action: 'Previous frame', isGlobal: true},
+    {key: '→', action: 'Next frame', isGlobal: true},
+    {key: 'Shift + ←', action: 'Reset to first frame', isGlobal: true},
+    {key: 'Shift + →', action: 'Seek to last frame', isGlobal: true},
+    {key: 'm', action: 'Toggle audio', isGlobal: true},
```

Comment:
Using uppercase letters would be more readable:
```suggestion
    {key: 'M', action: 'Toggle audio', isGlobal: true},
```